downstream/modules/platform/con_storage-options-for-operator-installation-on-ocp.adoc

```diff
@@ -0,0 +1,14 @@
+[id="con_storage-options-for-operator-installation-on-ocp_{context}"]
+= Storage options for {OperatorPlatform} installation on {OCP}
+
+If you are using file-based storage and your installation scenario includes {HubName}, you must ensure that your storage option for {OperatorPlatform} has been set to `ReadWriteMany`.
```

Comment:
```suggestion
If you are using file-based storage and your installation scenario includes {HubName}, ensure that the storage option for {OperatorPlatform} is set to `ReadWriteMany`.
```

Reply:
Out of scope for this PR, I know, but I'm curious whether there's a default setting for the storage option.

Reply:
@jself-sudoku It's not essential, but it might be a good idea in this case, because both option names begin with `ReadWrite`.

Reply:
Thanks, @ariordan-redhat. I have updated the intro sentence to show the default setting.